### DISCUSSION TITLE
Option to use Plex Music video format

### DISCUSSION THREAD

**Discussion by julianrinaldi** [RandomNinjaAtk/docker-lidarr-extended#87] [Ideas]
I think I saw this asked before, but having an option to have Lidarr Extended put the music video files in the same folders as the album/song would be very useful. I know the current implementation is intended to be more "compatible" but if it doesn't add them so that Plex finds them it's not useful at all for me.

**Comment by RandomNinjaAtk** [comment]
Plex finds the videos just fine, from my personal usage. If your having trouble with Plex, make sure you've configured the global folder and performed a full refresh on the artist.

Edit: Wiki details: https://github.com/RandomNinjaAtk/docker-lidarr-extended/wiki/Extended-Video-Script-Information
